fast64_internal/f3d/f3d_material_presets.py

```diff
@@ -6380,25 +6380,29 @@
 f3d_mat.presetName = 'Oot Water Mult Specular Fresnel'
 """
 
+homebrew_and_oot = {
+    "shaded_environment_mapped": shaded_environment_mapped,
+    "shaded_environment_mapped_transparent": shaded_environment_mapped_transparent,
+    "shaded_multitexture_lerp": shaded_multitexture_lerp,
+    "shaded_multitexture_lerp_transparent": shaded_multitexture_lerp_transparent,
+    "shaded_solid": shaded_solid,
+    "shaded_multitexture_lerp_transparent_vertex_alpha": shaded_multitexture_lerp_transparent_vertex_alpha,
+    "shaded_solid_transparent": shaded_solid_transparent,
+    "shaded_texture": shaded_texture,
+    "shaded_texture_cutout": shaded_texture_cutout,
+    "shaded_texture_transparent": shaded_texture_transparent,
+    "shaded_texture_transparent_vertex_alpha": shaded_texture_transparent_vertex_alpha,
+    "unlit_texture": unlit_texture,
+    "unlit_texture_cutout": unlit_texture_cutout,
+    "unlit_texture_transparent": unlit_texture_transparent,
+    "vertex_colored_texture": vertex_colored_texture,
+    "vertex_colored_texture_cutout": vertex_colored_texture_cutout,
+    "vertex_colored_texture_transparent": vertex_colored_texture_transparent,
+}
 material_presets = {
+    "homebrew": homebrew_and_oot,
     "oot": {
-        "oot_shaded_environment_mapped": oot_shaded_environment_mapped,
-        "oot_shaded_environment_mapped_transparent": oot_shaded_environment_mapped_transparent,
-        "oot_shaded_multitexture_lerp": oot_shaded_multitexture_lerp,
-        "oot_shaded_multitexture_lerp_transparent": oot_shaded_multitexture_lerp_transparent,
-        "oot_shaded_multitexture_lerp_transparent_vertex_alpha": oot_shaded_multitexture_lerp_transparent_vertex_alpha,
-        "oot_shaded_solid": oot_shaded_solid,
-        "oot_shaded_solid_transparent": oot_shaded_solid_transparent,
-        "oot_shaded_texture": oot_shaded_texture,
-        "oot_shaded_texture_cutout": oot_shaded_texture_cutout,
-        "oot_shaded_texture_transparent": oot_shaded_texture_transparent,
-        "oot_shaded_texture_transparent_vertex_alpha": oot_shaded_texture_transparent_vertex_alpha,
-        "oot_unlit_texture": oot_unlit_texture,
-        "oot_unlit_texture_cutout": oot_unlit_texture_cutout,
-        "oot_unlit_texture_transparent": oot_unlit_texture_transparent,
-        "oot_vertex_colored_texture": oot_vertex_colored_texture,
-        "oot_vertex_colored_texture_cutout": oot_vertex_colored_texture_cutout,
-        "oot_vertex_colored_texture_transparent": oot_vertex_colored_texture_transparent,
+        key: value + 'f3d_mat.presetName = "OOT " +  f3d_mat.presetName' for key, value in homebrew_and_oot.items()
```

Comment:
```suggestion
        f"oot_{key}": value + 'f3d_mat.presetName = "Oot " +  f3d_mat.presetName' for key, value in homebrew_and_oot.items()
```
without the `f"oot_{key}"` change, this is adding presets and removing the `oot_` prefixed ones
This behaves when updating fast64 as just writing out the "new" presets and keeping the "old" ones.
Basically, it duplicates the presets:
```
dragorn421@dragorn421pc:~/.config/blender/4.1/scripts/presets/f3d/oot$ ls
oot_shaded_environment_mapped.py                          oot_unlit_texture.py                                  shaded_solid_transparent.py
oot_shaded_environment_mapped_transparent.py              oot_unlit_texture_transparent.py                      shaded_texture_cutout.py
oot_shaded_multitexture_lerp.py                           oot_vertex_colored_texture_cutout.py                  shaded_texture.py
oot_shaded_multitexture_lerp_transparent.py               oot_vertex_colored_texture.py                         shaded_texture_transparent.py
oot_shaded_multitexture_lerp_transparent_vertex_alpha.py  oot_vertex_colored_texture_transparent.py             shaded_texture_transparent_vertex_alpha.py
oot_shaded_solid.py                                       __pycache__                                           unlit_texture_cutout.py
oot_shaded_solid_transparent.py                           shaded_environment_mapped.py                          unlit_texture.py
oot_shaded_texture_cutout.py                              shaded_environment_mapped_transparent.py              unlit_texture_transparent.py
oot_shaded_texture.py                                     shaded_multitexture_lerp.py                           vertex_colored_texture_cutout.py
oot_shaded_texture_transparent.py                         shaded_multitexture_lerp_transparent.py               vertex_colored_texture.py
oot_shaded_texture_transparent_vertex_alpha.py            shaded_multitexture_lerp_transparent_vertex_alpha.py  vertex_colored_texture_transparent.py
oot_unlit_texture_cutout.py                               shaded_solid.py
```
the "old" set is oot_-prefixed, the "new" one is not.

The two sets also show in the UI, one being "Oot "-prefixed and the other not
![image](https://github.com/Fast-64/fast64/assets/5306619/abc4a97e-c238-490d-8408-d32fc6b581c1)

(I am guessing the names in the list are generated by `bpy.types.Menu.path_menu` from the file names, like `abc_def.py` -> "Abc Def" ?)

----

the other OOT -> Oot change is not necessary but for consistency with current names.